DroppedText-20220601.txt

```diff
@@ -0,0 +1 @@
+D:\git_personal\AspNetCore.Diagnostics.HealthChecks\.github\workflows\healthchecks_aws_secretsmanager_cd_preview.yml
```

Comment:
remove this file

Reply:
I deleted both the secrets and system ones.

Reply:
Still here.

Reply:
Yes sorry still there. Should be deleted? Asked on below comment about that.

Reply:
I'm 100% sure that `DroppedText-20220601.txt` file should be deleted.